ffmpeg/ffmpeg.go

```diff
@@ -372,6 +382,17 @@ func (t *Transcoder) Transcode(input *TranscodeOptionsIn, ps []TranscodeOptions)
 			Frames: int(r.frames),
 			Pixels: int64(r.pixels),
 		}
+		//add dnn result
+		dnnresult := ""
```

Comment:
We can declare this in the following if block since we don't need the var unless we're in the if block.